tensorflow/java/src/main/java/org/tensorflow/Tensor.java

```diff
@@ -224,6 +224,74 @@
     return t;
   }
 
+  /**
+   * Create a {@link Integer} Tensor with data from the given array.
+   *
+   * <p>Creates a Tensor with the given shape by copying elements from the array into the tensor.
+   * On Android this method is preferred over {@link #create(long[], java.nio.IntBuffer) create}
+   * due to Android's ByteBuffer performance issues.
+   *
+   * @param shape the tensor shape.
+   * @param data an array containing the tensor data.
+   * @throws IllegalArgumentException If the tensor shape is not compatible with the array
+   */
+  public static Tensor<Integer> create(long[] shape, int[] data) {
+    Tensor<Integer> t = allocateForBuffer(DataType.INT32, shape, data.length);
+    setArray(t.nativeHandle, data);
```

Comment:
error: cannot find symbol
    setArray(t.nativeHandle, data);
              ^
  symbol:   variable nativeHandle
  location: variable t of type Tensor<Integer>